tests/auth_tests/test_tokens.py

```diff
@@ -32,6 +32,23 @@ def test_10265(self):
         tk2 = p0.make_token(reload)
         self.assertEqual(tk1, tk2)
 
+    def test_26615(self):
+        """
+        Ensure that the token generated for a user will fail if that user
+        changes email addresses.
+        """
+        # See ticket #26615
+        username = 'changeemailuser'
+        user = User.objects.create_user(
+            username, 'test4@example.com', 'testpw')
```

Comment:
Style: We can write this in one line like the other create_user examples in test_tokens.py.